x/auth/client/rest/decode.go

```diff
@@ -47,8 +47,9 @@ func DecodeTxRequestHandlerFn(clientCtx client.Context) http.HandlerFunc {
 			return
 		}
 
-		stdTx, ok := convertToStdTx(w, clientCtx, txBytes)
-		if !ok {
+		stdTx, err := convertToStdTx(w, clientCtx, txBytes)
+		if err != nil {
+			// Error is already returned by convertToStdTx.
```

Comment:
This just tells me we need better logging...actually any logging at all in our HTTP APIs

Reply:
Also, how is this error used exactly?

Reply:
The error is already outputted by the `w http.ResponseWriter`, see for example https://github.com/cosmos/cosmos-sdk/pull/7730/files#diff-69c02b40843f1d63e016e4b695cf245b64d7d883deb362eeeb0f79e30f796e73R73